index.js

```diff
@@ -22,11 +22,30 @@ const cacheAndBufferLayer = require('./lib/CacheAndBufferLayer');
 const channels = require('channels');
 const util = require('util');
 
-const defaultLogger = {
-  debug: () => {},
-  info: () => {},
-  error: () => {},
-  warn: () => {},
+// Different logging libraries have different ways to check if a particular log level is enabled.
```

Comment:
Afaik we only support log4js but normalizing this is a bonus :)  Are we aware of this as a requirement tho?

Reply:
There is no documentation saying what kind of logger we expect, and I'm not sure how many projects use ueberdb2 other than Etherpad. I can remove this logic as YAGNI if you would prefer.